codecparsers/vp9parser.c

```diff
@@ -517,9 +517,10 @@ static void set_default_lf_deltas(Vp9Parser* parser)
   priv->mode_deltas[1] = 0;
 }
 
-static void setup_past_independence(Vp9Parser* parser)
+static void setup_past_independence(Vp9Parser* parser, Vp9FrameHdr* const frame_hdr)
 {
   set_default_lf_deltas(parser);
+  frame_hdr->ref_frame_sign_bias[0] = 0;
```

Comment:
why wen need this  since we have "  memset(frame_hdr, 0, sizeof(*frame_hdr));" in vp9_parse_frame_header